source/blog/2019-05-11-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,181 @@
+---
+title: RSpec Rails 4.0 has been released!
+author: Benoit Tigeot, Jon Rowe and Phil Pirozhkov
+---
+
+RSpec Rails 4.0 has been released recently! Given our commitment to
+[semantic versioning](http://semver.org/), this update comes with 
+breaking changes for people that are using Rails below 4.2. If we 
+did introduce any bugs, please let us know, and we'll get 
+a patch release out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes 258 commits and 56
+merged pull requests from 27 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+Support for Rails 6, support for testing ActionCable, ActiveMailbox
+
+### Breaking changes
+
+RSpec Rails 4 supports only Rails 5 and 6. This is in line with our
+new versioning policy which keeps major versions of RSpec Rails in 
+lock step with versions of Rails.
+
+This means that we no longer support versions of Ruby below 2.2
+in line with the supported Ruby versions of Rails 5.
+
+We have not restricted the version of Rails/Ruby for this version to
```

Comment:
[We do now](https://github.com/rspec/rspec-rails/pull/2338/files#diff-73c25cf975ccb4c89d8659202506c353R14), so probably the whole paragraph changes it meaning. Maybe something like:
"We soft-support Rails 4.2 providing a smoother upgrade path, but we highly recommend using RSpec Rails 4.0 with Rails 5.0 and up. If you decide to stick with Rails 4.2 for a while, it would be better to use RSpec Rails version 3".

Reply:
I think your right but the point still stands and I'm happy with the rest of this paragraph
```suggestion
We have only restricted the version of Rails for this version
```